### DISCUSSION TITLE
cbind add column in jax

### DISCUSSION THREAD

**Discussion by geoHeil** [jax-ml/jax#5974] [Q&A] [answered]
The `cbind` function in R is implemented in numpy as:

import numpy as np
np.c_[[1,2],[3,4]]

However, `np.c_` is not available in JAX.

How can I append the column instead when using JAX?

**Comment by geoHeil** [comment]
```
jnp.append(jnp.array([1,2]),55)
```

is nicely appending to the single dimension vector.

However, for:

```
jnp.append(jnp.array([1,2]),jnp.array([3,4]))
```

is appending to the vector as rows (axis=0), it fails to append to axis=1. How can I get c (=column bind) instead of rbind?

**Comment by hawkinsp** [comment, marked as answer]
Try `jnp.column_stack([[1,2],[3,4]])`

There's also an open issue to implement `jnp.c_` (https://github.com/google/jax/issues/5850)

**Reply by geoHeil** [reply]
Great Thanks
